api/include/opentelemetry/context/runtime_context.h

```diff
@@ -65,7 +48,7 @@ class RuntimeContextStorage
    * Set the current context.
    * @param the new current context
    */
-  virtual Token Attach(Context context) noexcept = 0;
+  virtual nostd::unique_ptr<Token> Attach(Context context) noexcept = 0;
```

Comment:
since this is now a unique pointer, can you add more documentation about its expected lifecycle?

E.g. how long can we hold onto this, and is it always safe to reference it?

Reply:
I added some documentation saying this always returns a valid pointer (never a `nullptr`).

According to smart pointer semantics, returning a `unique_ptr` passes ownership to the caller, who decides about the lifetime.